p4-16/spec/P4-16-spec.mdk

```diff
@@ -3732,39 +3732,39 @@ finding this information in a section dedicated to type `varbit`.
 
 P4 provides a limited set of casts between types. A cast is written
 `(t) e`, where `t` is a type and `e` is an expression. Casts are only
-permitted on base types and derived types introduced by `typedef`, `type`, and `enum`. While this design is arguably more onerous
-for programmers, it has several benefits:
+permitted on base types and derived types introduced by `typedef`, `type`, and `enum`.
+While this design is arguably more onerous for programmers, it has several benefits:
 
 - It makes user intent unambiguous.
 - It makes the costs associated with converting numeric values
-  explicit. Implementing certain casts involve sign-extensions, and
-  thus can require significant computational resources on some
-  targets.
+  explicit. Implementing certain casts involve sign extensions,
+  and thus can require significant computational resources on some targets.
 - It reduces the number of cases that have to be considered in the P4
   specification. Some targets may not support all casts.
 
 ### Explicit casts
 
 The following casts are legal in P4:
 
-- `bit<1> <-> bool`: converts the value `0` to `false`, the value `1` to `true`, and vice versa.
-- `int -> bool`: only if the `int` value is `0` (converted to `false`) or `1` (converted to `true`)
-- `int<W> -> bit<W>`: preserves all bits unchanged and reinterprets negative
+- `bit<1> ↔ bool`: converts the value `0` to `false`, the value `1` to `true`, and vice versa.
```

Comment:
This arrow is unreadable at least in the markup file. Also, latex is not necessarily Unicode-friendly, did you check to see how it looks in the pdf?

Reply:
AFAIK this spec is not written in the LᴀTᴇX language, but rather in the “Madoko” language.

Reply:
Yes, but it produces two consumables: one in pdf, through latex, and one in html.
The question is about how these look.

Reply:
After some testing [using <https://hub.docker.com/r/p4lang/madoko-debian>], I found that Unicode support was broken by default.

After extensive experimentation, I found that only 1 out of 4 different ways of trying to fix it actually _works_.

![image](https://user-images.githubusercontent.com/3038439/166590613-8ebb44b3-1b98-4151-8cc0-68d352feade0.png)

![image](https://user-images.githubusercontent.com/3038439/166590539-4cbe0dd7-7cb4-4156-a466-2d1ddd1a2be0.png)

Reply:
### Before (from r1.2.2)
![image](https://user-images.githubusercontent.com/3038439/166604563-6637e9b2-4db1-4093-9388-18c3c4fe21bc.png)

### After (from my latest WIP)
![image](https://user-images.githubusercontent.com/3038439/166604406-09603e39-36be-4cdb-9c01-7307d3cceb59.png)